source/_patterns/_mixins.scss

```diff
@@ -280,6 +280,18 @@ $light-attention-color: #ff0000;
   }
 }
 
+@keyframes pulsatingLiveAnimation {
```

Comment:
wollen wir die animations in ein separates file auslagern? gehören ja eigentlich nicht in die mixins.